avaframe/in2Trans/shpConversion.py

```diff
@@ -57,11 +57,11 @@ def SHP2Array(infile, defname=None):
     infile = pathlib.Path(infile)
     # set defaults for variables
     layername = None
-    d0 = None
+    relTh = None
     rho = None
     sks = None
     iso = None
-    d01 = []
+    relTh1 = []
```

Comment:
What is this needed for?

Reply:
this is the list where the thickness values of the individual features are appended i renamed it to thicknessList and moved it to the respective list initialisation for the name